unit-8/menu-hover-effect/scss/style.scss

```
@@ -0,0 +1,256 @@
+                opacity: 0;
+                visibility: hidden;
+                transition: all 300ms linear;
+                z-index: 1000;
+                left:280px;
+                top: 0px;
+                border-left: 4px solid $color-orange;
+
+                &::before {
+                    content: " ";
+                    position: absolute;
+                    top: 25px;
+                    left: -9px;
+                    border-right: 5px solid $color-orange;
+                    border-bottom: 5px solid transparent;
+                    border-top: 5px solid transparent;
+                }
+
+                li {
+                    a {
+                        padding: 12px 20px;
+                        text-align: left;
+                        border: 0;
+                        border-bottom: 1px solid $color-gray;
+                        height: auto;
+
+                        i {
+                            font-size: 16px;
+                            display: inline-block;
+                            margin: 0 10px 0 0;
+                        }
+                    }
+
+                    &:hover > ul {
+                        display: block;
+                        opacity: 1;
+                        visibility: visible;
+                        left:250px;
+                        top: 0px;
+                        left: 200px;
+                    }
+
+                    ul {
+                        position: absolute;
+                        height: auto;
+                        min-width: 200px;
+                        padding: 0;
+                        margin: 0;
+                        background: $color-bg-ul;
+                        opacity: 0;
+                        visibility: hidden;
+                        transition: all 300ms linear;
+                        z-index: 1000;
+                        left: 230px;
+                        top: 0;
+                        border: 0;
+                        border-left: 4px solid $color-orange;
+
+                        &::before {
+                            content: " ";
+                            position: absolute;
+                            top: 15px;
+                            left: -9px;
+                            border-right: 5px solid $color-orange;
+                            border-bottom: 5px solid transparent;
+                            border-top: 5px solid transparent;
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+@-webkit-keyframes moveFromLeft {
+    from {
+        opacity: 0;
+        transform: translateX(150%);
+    }
+    to {
+        opacity: 1;
+        transform: translateX(0%);
+    }
+}
+
+.container {
+    .menu {
+        li {
+            a {
+                i,
+                span,
+                p {
+                    position: relative;
+                }
+            }
+        }
+        li:hover a{
```

```suggestion
        li:hover a {
```
trước `{` hãy thêm 1 dấu cách